job-descriptions/engineering-manager-product.md

```diff
@@ -0,0 +1,41 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Engineering Manager - Product
+
+We are looking for an experienced engineering leader to manage a cross-functional engineering team of ~3-8 engineers that is responsible for one of our products (e.g., [campaigns](https://about.sourcegraph.com/handbook/engineering/campaigns), [code intelligence](https://about.sourcegraph.com/handbook/engineering/code-intelligence), code host integrations). You will partner with our [product team](https://about.sourcegraph.com/handbook/product) to define team goals, and then develop a roadmap to achieve those goals. Our product is very technical, so we expect you to be capable of doing a non-zero amount of development work that is not on the critical path to keep your technical knowledge up-to-date.
+
+## Qualifications
+
+- Experience leading teams to achieve a product objective.
```

Comment:
We've had recent conversations that make me believe this role now would require greater engineering experience. I worry this bullet doesn't capture the depth of technical competency we expect. I'm not remembering what our final decision was here, but I think it is that this person would be the partner to product to help drive engineering initiatives. They would need to be able to collaborate (and sometimes contribute) to technical efforts.

Reply:
Would lifting "- Experience working with one or more of technologies that we use (e.g., Go, TypeScript, Kubernetes)." address your concern?

If I did that, should I move "Experience leading teams to achieve a product objective." to "Nice-to-haves"?

Reply:
Yes, that would help, I would not move leading teams down, don't we want both?